src/emulator/loggingEmulator.ts

```diff
@@ -11,6 +11,20 @@ export interface LoggingEmulatorArgs {
   host?: string;
 }
 
+export interface LogData {
```

Comment:
nit: Should this be called `LogData` or `LogMetadata`?

Reply:
no, logdata because it contains both the `user` data and the `metadata`